src/components/canrawview/canrawview.cpp

```diff
@@ -18,8 +21,20 @@ CanRawView::~CanRawView()
 {
 }
 
+void CanRawView::simulationStarted()
+{
+    timer->restart();
+    simStarted = true;
+}
+
+void CanRawView::simulationStopped()
+{
+    simStarted = false;
```

Comment:
http://doc.qt.io/qt-5/qelapsedtimer.html

QElapsedTimer doesn't have stop function, restart method starts it from 0.